packages/terra-table/src/subcomponents/ColumnHeader.jsx

```diff
@@ -84,6 +84,8 @@ const ColumnHeader = (props) => {
   return (
     <thead>
       <tr
+        aria-rowindex={1}
+        data-row-id={`${tableId}-header-row`}
```

Comment:
There should be Jest updates based on this change.  You should be testing that these value exist, as expected.

Reply:
It appears you missed validation for the aria-rowindex.